src/main/java/com.gigssandbox/StringToCalendarParser.java

```diff
@@ -0,0 +1,64 @@
+package com.gigssandbox;
+
+import com.gigssandbox.exceptions.DateParsingException;
+import com.gigssandbox.exceptions.IncorrectDateValueException;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+class StringToCalendarParser {
```

Comment:
Why not use standard Java date utils?

e.g.
```
String string = "January 2, 2010";
DateFormat format = new SimpleDateFormat("MMMM d, yyyy", Locale.ENGLISH);
Date date = format.parse(string);
```

Reply:
In some place at the Stackoverflow I found that Calendar is more convenient to create specific date than Date; also I need to verify that entered string with date content is correct or not, in this case Calendar supports, for example, 32.01.2010, while Date will show 1.02.2010 iat the same situation, but for me both answers are incorrect, and I should check input for correctness, this class does only this responsibility